risc0/zkvm/src/lib.rs

```diff
@@ -84,6 +82,7 @@ pub use self::host::{
 #[cfg(not(target_os = "zkvm"))]
 pub use self::host::{
     control_id::POSEIDON_CONTROL_ID,
+    groth16::{Groth16Proof, Groth16Seal},
```

Comment:
I might suggest not putting `Groth16Proof` and `Groth16Seal` in the global namespace. They feel like lower level details.
```suggestion
    groth16
```

Reply:
This does not work as then we would need to make the `groth16` mod public, something like
```
#[cfg(not(target_os = "zkvm"))]
pub mod groth16;
```

but then it would be the exception. I think keeping it the way it is now is more consistent with the way we are controlling what to re-export